src/main/resources/org/jenkinsci/plugins/workflow/cps/RunWrapperBinder/help.jelly

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core">
-    <p>
-        The <code>currentBuild</code> variable may be used to refer to the currently running build.
-        It is an object similar to that documented for the return value of the <code>build</code> step (when <code>wait</code> is enabled).
```

Comment:
This quasi-link was the main motivation behind the issue. Now the actual content is visible here.